examples/asana-googlecalendar/index.js

```diff
@@ -0,0 +1,183 @@
+const express = require("express");
+const asana = require("asana");
+const { google } = require("googleapis");
+const everyauth = require("@fusebit/everyauth-express");
+const moment = require("moment");
+const { v4: uuidv4 } = require("uuid");
+const cookieSession = require("cookie-session");
+const mustacheExpress = require("mustache-express");
+
+const app = express();
+const port = 3000;
+
+app.engine("mustache", mustacheExpress());
+app.set("view engine", "mustache");
+app.set("views", __dirname + "/views");
+app.use(express.json());
+app.use(express.urlencoded());
+app.use(cookieSession({ name: "session", secret: "secret" }));
+
+app.get("/", (req, res) => {
+  if (req.session.asanaUserId && req.session.googleUserId) {
+    return res.redirect(`/tasks`);
+  }
+  return res.render("index", { userId: uuidv4() });
```

Comment:
This will generate a unique userId for every single page refresh.  Is that what you want?